docs/deployment/resource-providers/standalone/docker.md

```diff
@@ -344,64 +389,36 @@ as described in [how to run the Flink image](#how-to-run-flink-image).
     ENV VAR_NAME value
     ```
 
-    **Commands for building**:
+  **Commands for building**:
 
     ```sh
     docker build -t custom_flink_image .
     # optional push to your docker image registry if you have it,
     # e.g. to distribute the custom image to your cluster
     docker push custom_flink_image
     ```
-  
-### Enabling Python
 
-To build a custom image which has Python and Pyflink prepared, you can refer to the following Dockerfile:
-{% highlight Dockerfile %}
-FROM flink
-
-# install python3 and pip3
-RUN apt-get update -y && \
-apt-get install -y python3.7 python3-pip python3.7-dev && rm -rf /var/lib/apt/lists/*
-RUN ln -s /usr/bin/python3 /usr/bin/python
 
-# install Python Flink
-RUN pip3 install apache-flink
-{% endhighlight %}
-
-Build the image named as **pyflink:latest**:
-
-{% highlight bash %}
-sudo docker build -t pyflink:latest .
-{% endhighlight %}
-
-{% top %}
-
-## Flink with Docker Compose
+### Flink with Docker Compose
 
 [Docker Compose](https://docs.docker.com/compose/) is a way to run a group of Docker containers locally.
-The next chapters show examples of configuration files to run Flink.
+The next sections show examples of configuration files to run Flink.
 
-### Usage
+#### Usage
 
 * Create the `yaml` files with the container configuration, check examples for:
-    * [Session cluster](#session-cluster-with-docker-compose)
-    * [Job cluster](#job-cluster-with-docker-compose)
+  * [Application cluster](#app-cluster-yml)
+  * [Session cluster](#session-cluster-yml)
```

Comment:
I couldn't find the corresponding anchor points/yaml files. The targets of these links do not exist.

Reply:
I just realized that you used the collapsible code blocks. Reading through it right now, I guess, I prefer the Appendix approach we use in the Kubernetes documentation page.

Reply:
Okay, I'll revert the collapse-thing.
I don't think adding another indirection to a appending makes sense here, since this is already the bottom of the page.